src/main/core/sim_config.rs

```diff
@@ -342,8 +343,9 @@ fn build_process(proc: &ProcessOptions) -> anyhow::Result<Vec<ProcessInfo>> {
             .with_context(|| format!("Failed to parse arguments: {x}"))?,
     };
 
-    // perform shell expansion
-    let plugin = tilde_expansion(proc.path.to_str().unwrap());
+    // resolve path
+    let plugin = resolve_path(&proc.path)
+        .with_context(|| format!("Failed to resolve plugin path \"{:?}\"", proc.path))?;
```

Comment:
Please add an info-level log statement here with the resolved path. e.g.
`info!("Resolved process path {proc.path} to {plugin}")`